narwhals/_arrow/series.py

```diff
@@ -298,6 +299,17 @@ def std(self, ddof: int = 1) -> int:
 
         return pc.stddev(self._native_series, ddof=ddof)  # type: ignore[no-any-return]
 
+    def skew(self) -> float:
```

Comment:
Although it would end up returning a pyarrow scalar, I think we should keep the implementation with native methods, or you can reuse methods implemented, such as all elementary operations